### PR TITLE
#1718 changed dynamic link to postman sandbox api reference

### PR DESCRIPTION
**Pull request by stcalica**
Fix for #1718 
Changed the link in the docs to the Sandbox API reference. 

The only issue is I don't know how to do an anchor by id in Jekyell so I only did a hard link to the site's anchor on dynamic variables. 

If someone can find and show me an example of an anchor link on a header that'd be helpful and I can make that change.

### CODE REVIEW

File: v6/postman/environments_and_globals/variables.md [hunk 1]
```diff
@@ -107,13 +107,15 @@ Dynamic variables cannot be used in the Sandbox. You can only use them in the `
    *   `{{$timestamp}}`: Adds the current timestamp
    *   `{{$randomInt}}`: Adds a random integer between 0 and 1000
 
-   For a complete list of dynamic variables, refer to the section [Dynamic Variables List](/docs/postman/environments_and_globals/variables_list).
+   For a complete list of dynamic variables, refer to the section [Dynamic Variables List](https://learning.getpostman.com/docs/postman/scripts/postman_sandbox_api_reference/#dynamic-variables).
```
Comment (tristandenyer) [inline v6/postman/environments_and_globals/variables.md]: @stcalica You can leave off the domain `https://learning.getpostman.com`
Links can start with `/docs`, even the anchor tags.

File: v6/postman/environments_and_globals/variables.md [hunk 2]
```diff
@@ -107,13 +107,15 @@ Dynamic variables cannot be used in the Sandbox. You can only use them in the `
...
 
+
+https://learning.getpostman.com/docs/postman/scripts/postman_sandbox_api_reference/#dynamic-variables
```
Comment (tristandenyer) [inline v6/postman/environments_and_globals/variables.md]: Remove this line. It just gets printed as a string.

File: v6/postman/environments_and_globals/variables.md (hunk 1 above)
Comment (stcalica) [reply v6/postman/environments_and_globals/variables.md]: awesome! nice and simple, love it!

**Comment by stcalica** [comment]
@tristandenyer  can't merge this since I'm not authorized. Mind merging it for me?

**Pull request merged**